src/back/routes/badges.ts

```diff
@@ -72,3 +74,24 @@ async function revokeBadge(req: WithAuth): Promise<string> {
     return `Failed to revoke badges ${JSON.stringify(e)}`
   }
 }
+
+export type UploadResult = { status: ActionStatus; badgeCid: string; error?: string }
```

Comment:
Maybe we can move this to a new file `back/types/badges.ts`?

Reply:
I sent it to entities/badges/types so I can refactor them all together on next PR